src/services/kuma-api/KumaApi.ts

```diff
@@ -102,6 +108,10 @@ export default class KumaApi extends Api {
     return this.client.get(`/zoneingresses+insights/${name}`, { params })
   }
 
+  getZoneEgress({ name }: { name: string }, params?: any): Promise<ZoneEgress> {
+    return this.client.get(`/zoneegresses/${name}`, { params })
+  }
```

Comment:
Are we retrieving new data with this PR that we weren't retrieving before?

Reply:
Yes, we are. I did check these APIs exist and work as expected.

Reply:
Cool ty